### DISCUSSION TITLE
Curation Report Week of May 23-29

### DISCUSSION THREAD

**Discussion by susancoort** [wikipathways/GPML-Repo#71] [Curation reports]
Dear all,
My apologies for the late report.
- Welcomed a new user: VPencheva
- Added ontology tags to WP5238 (new pathway -> under construction)
- WP5235: Asked the user, via a discussion, to create a bibliography and not just list the references in the description. Gave this link: http://academy.wikipathways.org/stages/wp-add-literature/  
-  WP5215: "Needs work" tag added
- The search function works 

How do I know who is next? Should I keep the order on this list? https://github.com/wikipathways/academy/blob/gh-pages/curators/curators_list.xml. If so, then @andrawaag is next.
With kind regards,
Susan

**Comment by mkutmon** [comment]
@susancoort, yes, I think that is the correct file to check for the next person indeed.
